README.md

```diff
@@ -3,12 +3,40 @@
 This python project is a helper package that uses power analysis to calculate required sample size for any experiment.
 
 ## Script Usage Guide
```

Comment:
made some updates to this instruction considering BA might not have python on their computer somehow